pkg/provisioners/certcentral.go

```diff
@@ -31,8 +32,9 @@ type CertCentral struct {
 	name   string
 	client *certcentral.Client
 
-	validityYears,
-	organizationID int
+	validityDays,
+	validityYears *int
```

Comment:
I'd avoid this member declaration. Better to have a type side by side.
```suggestion
	validityDays *int
	validityYears *int
```

Reply:
I agree, I don't like it either. I just wanted to adhere to the code style from before, but will change it.